islandora_solr_config/islandora_solr_config.module

```diff
@@ -55,6 +55,15 @@ function islandora_solr_config_islandora_solr_secondary_display() {
       'description' => t("Show search results as RSS feed"),
       'logo' => '<img src="' . $path . '/images/rss.png" class="secondary-display-rss" alt="' . t("RSS Feed") . '">',
     ),
+    'atom' => array(
+      'name' => t('Atom'),
+      'module' => 'islandora_solr_config',
+      'file' => 'includes/atom_results.inc',
+      'class' => 'IslandoraSolrResultsAtom',
+      'function' => 'printAtom',
+      'description' => t("Show search results as an Atom feed"),
+      'logo' => '<img src="' . $path . '/images/rss.png" class="secondary-display-rss" alt="' . t("Atom Feed") . '">',
```

Comment:
This should have its own image and class.

Reply:
Any suggestions for an Atom image?

Reply:
Looks like in general, the same icon is used for both [Atom](https://en.wikipedia.org/wiki/Atom_(Web_standard)#/media/File:Feed-icon.svg) and [RSS](https://en.wikipedia.org/wiki/RSS#/media/File:Feed-icon.svg).

Reply:
Its the same, when Atom was invented bitmaps where just a dream.... I would either go with the same or duplicate the file and give it another nae so people could eventually decide for a replacement?